plugins/available/functional-test-plugin/functionalTestPlugin.js

```diff
@@ -104,10 +104,10 @@ class FunctionalTestPlugin {
     this.pipes['plugin-functional-test-plugin:testPipesReturn'] =
       async name => `Hello, ${name}`;
 
-    // Pipe declared with a function name
+    // Pipe declared with a function name ======================================
```

Comment:
(nitpicking / for another PR) when I see that kind of thing, this triggers a reflex telling me that the file is too large, and that the code should be split to make it more readable